docs/docs/images/github-round.svg

```diff
@@ -0,0 +1,6 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<svg xmlns="http://www.w3.org/2000/svg" xmlns:xlink="http://www.w3.org/1999/xlink" width="46px" height="46px" viewBox="0 0 46 46" version="1.1">
+<g id="surface1">
+<path style=" stroke:none;fill-rule:evenodd;fill:rgb(14.117647%,16.078431%,18.431373%);fill-opacity:1;" d="M 22.929688 0 C 10.25 0 0 10.542969 0 23.582031 C 0 34.007812 6.566406 42.832031 15.679688 45.957031 C 16.820312 46.191406 17.238281 45.449219 17.238281 44.824219 C 17.238281 44.277344 17.199219 42.402344 17.199219 40.449219 C 10.820312 41.855469 9.492188 37.640625 9.492188 37.640625 C 8.464844 34.90625 6.949219 34.203125 6.949219 34.203125 C 4.859375 32.757812 7.097656 32.757812 7.097656 32.757812 C 9.414062 32.914062 10.628906 35.179688 10.628906 35.179688 C 12.679688 38.769531 15.984375 37.757812 17.3125 37.132812 C 17.503906 35.609375 18.109375 34.554688 18.753906 33.96875 C 13.667969 33.421875 8.316406 31.390625 8.316406 22.332031 C 8.316406 19.757812 9.226562 17.648438 10.667969 16.007812 C 10.441406 15.421875 9.644531 13 10.894531 9.761719 C 10.894531 9.761719 12.832031 9.136719 17.199219 12.183594 C 19.066406 11.664062 20.996094 11.402344 22.929688 11.402344 C 24.867188 11.402344 26.839844 11.675781 28.664062 12.183594 C 33.03125 9.136719 34.964844 9.761719 34.964844 9.761719 C 36.21875 13 35.421875 15.421875 35.195312 16.007812 C 36.675781 17.648438 37.546875 19.757812 37.546875 22.332031 C 37.546875 31.390625 32.195312 33.382812 27.070312 33.96875 C 27.90625 34.710938 28.625 36.117188 28.625 38.34375 C 28.625 41.503906 28.589844 44.042969 28.589844 44.824219 C 28.589844 45.449219 29.007812 46.191406 30.144531 45.957031 C 39.257812 42.832031 45.824219 34.007812 45.824219 23.582031 C 45.863281 10.542969 35.574219 0 22.929688 0 Z M 22.929688 0 "/>
+</g>
+</svg>
```

Comment:
Same here, this can be optimized:
```suggestion
<svg xmlns="http://www.w3.org/2000/svg" width="46" height="46">
	<path d="M22.93 0C10.25 0 0 10.543 0 23.582c0 10.426 6.566 19.25 15.68 22.375 1.14.234 1.558-.508 1.558-1.133 0-.547-.039-2.422-.039-4.375-6.379 1.406-7.707-2.808-7.707-2.808-1.027-2.735-2.543-3.438-2.543-3.438-2.09-1.445.149-1.445.149-1.445 2.316.156 3.53 2.422 3.53 2.422 2.052 3.59 5.356 2.578 6.684 1.953.192-1.524.797-2.578 1.442-3.164-5.086-.547-10.438-2.578-10.438-11.637 0-2.574.91-4.684 2.352-6.324-.227-.586-1.023-3.008.227-6.246 0 0 1.937-.625 6.304 2.422a21.386 21.386 0 0 1 5.73-.782c1.938 0 3.91.274 5.735.782 4.367-3.047 6.3-2.422 6.3-2.422 1.255 3.238.458 5.66.231 6.246 1.48 1.64 2.352 3.75 2.352 6.324 0 9.059-5.352 11.05-10.477 11.637.836.742 1.555 2.148 1.555 4.375 0 3.16-.035 5.699-.035 6.48 0 .625.418 1.367 1.555 1.133 9.113-3.125 15.68-11.95 15.68-22.375C45.862 10.543 35.574 0 22.93 0Zm0 0" style="stroke:none;fill-rule:evenodd;fill:#24292f;fill-opacity:1"/>
</svg>
```